src/Raven.Server/Documents/Sharding/Handlers/Processors/Streaming/ShardedStreamingHandlerProcessorForGetDocs.cs

```diff
@@ -39,7 +39,7 @@ public ShardedStreamingHandlerProcessorForGetDocs([NotNull] ShardedDatabaseReque
                     OrderDocumentsById(streams, continuationToken) : 
                     OrderDocumentsByLastModified(streams, continuationToken);
 
-                await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandler.ResponseBodyStream(), token.Token))
+                await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandler.ResponseBodyStream()))
```

Comment:
Here we're in a streaming handled so potentially we're dealing with large responses. We no longer pass the cancellation token. How can we stop processing / writing the large response then?

Reply:
We should add that in the body.